mlir/test/e2e/Resnext101Config.toml

```diff
@@ -13,7 +13,7 @@ values = ["-fil_layout=gkcyx -in_layout=ngchw -out_layout=ngkhw", "-fil_layout=g
 
 [[axis]]
 name = "data type"
-values = ["f32"]
+values = ["f32", "bf16"]
```

Comment:
I'm not sure we want to enable here since this is f32-only unlike the rest of them?

Reply:
> I'm not sure we want to enable here since this is f32-only unlike the rest of them?

Removed